Bottom: 8310e1bb41a021b1a95d280143e66b5a91812187
Top:    495e619b44556b873c9148dbe020169e38479753
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-02 09:51:44 -0700

Refresh of allow-arbitrary-python-streams

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
index 377c6dadb1a..d1651498387 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
@@ -1222,7 +1222,7 @@ llvm::Expected<FileSP> PythonFile::ConvertToFile(bool borrowed) {
   if (borrowed) {
     // In this case we we don't need to retain the python
     // object at all.
-    file_sp = NativeFile::make_shared(fd, options.get(), false);
+    file_sp = std::make_shared<NativeFile>(fd, options.get(), false);
   } else {
     file_sp = std::static_pointer_cast<File>(
         std::make_shared<SimplePythonFile>(fd, options.get(), *this, borrowed));
